### DISCUSSION TITLE
[UCF] Several short videos showing how to install/use Obojobo

### DISCUSSION THREAD

**Discussion by zachberry** [ucfopen/Obojobo#1679] [Ideas]
Thinking of these videos (Ideally 0:30-2 min long)

* Overview of what it is (show student experience mostly, talk about gradebook sync)
* How to install obo next in your course
* How to create your first module
* How to embed your module in your course (graded vs non-graded)
* How to make your module cool (add images, videos, etc)
* How to work assessment (attempts, buttons, pre-test page, post-test page, question banks)
* Advanced - how to do pretests and threshold scoring stuff

**Comment by iturgeon** [comment]
This is cool. I was recently watching a few of Github's videos like this and I think it's a decent example to draw from.  

https://youtu.be/G_TP-2cRypU

As we're just sorta starting out on this idea it's probably totally best to avoid going overboard w/ production effort. If we can keep them simple to make we're more likely to continue making them.

I love the idea of having the people behind new features introducing it to everyone 👍
